package-managers/nuget.md

```diff
@@ -6,10 +6,63 @@ TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-n
 
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
+### Find our feed URL
+
+#### Private feed
+
+Private feeds are only accessible to the feed owner, and to whomever the feed owner granted sufficient privileges. When we need to connect to a private feed we don't own, we may need to ask the feed owner to invite us to the feed.
+
+> ### Hint
+> We can find out how to invite users [here](../inviting-users-to-a-feed.md)
+
+![Feed details](assets/feed-details.png)
+
+> ### Hint
+> We support [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider), the [NuGet CLI](#nuget-cli), [dotnet CLI](#dotnet-cli) and [Paket](#paket).
+
+#### Public feed
+
+There are two types of public feeds we can connect to. A public feed we own and one from the gallery. The only real difference here, is where to find the URL of the feed.
+
+MyGet feeds have multiple endpoints, and each one is specific to a particular package type and protocol. Pick the NuGet endpoint URL that matches the protocol version you want to use (e.g. NuGet v3), and use that to configure the feed endpoint in your favorite NuGet client.

```

Comment:
Maybe add one minor sentence: `Let's go over how to connect to these.`
In terms of order, preferably first user's own feeds and then second gallery feed?